### ISSUE TITLE
Installation wizard - Updating of the welcome page

### ISSUE THREAD

**Issue by crsfalzon** [SimplyVC/panic#195]
### User Story

As a node operator, I want to have a welcome page identifying the process that I will need to go through to set up monitoring for a sub-chain, so that I am aware of the required steps for config completion. 

### Description

This ticket is an adjustment for #45. Since the review page will not be implemented during this iteration, the Review step and its explanation are to be omitted for the time being.
![Remove review entry from Welcome page](https://user-images.githubusercontent.com/89395527/167349615-895f6eb5-c1e4-4a87-b316-ebe7c368a71a.PNG)

### Requirements

- Remove the Review section from the page, including the corresponding icon and explanation (see image above).
- Adjust the styling of the page as needed.

### Blocked by

Not dependent on another ticket to be tackled.

### Acceptance criteria

**Scenario**: Nope operator is on the installation wizard's welcome page
**Given**: This is the node operator's first use of PANIC
**When**: He is on the welcome page
 **Then**: The Review section is not visible and the Alerts step is followed by Feedback
